### ISSUE TITLE
If ProM and glBars is not built make breaks

### ISSUE THREAD

**Issue by guysoft** [DISTRHO/DPF-Plugins#10]
```
# ProM (needs OpenGL + ProjectM)
```

And later:
```
cp -r bin/ProM             /usr/local/bin/
/bin/cp: cannot stat 'bin/ProM': No such file or directory
Makefile:114: recipe for target 'install' failed
```

Source:
https://github.com/DISTRHO/DPF-Plugins/blob/master/Makefile#L48

Full log:
http://os.zynthian.org:8080/job/ZynthianOS/208/parsed_console/job/ZynthianOS/208/parsed_console/log_content.html#WARNING1

Bug in Zynthian-sys
https://github.com/zynthian/zynthian-sys/issues/74

Our build script:
https://github.com/zynthian/zynthian-sys/blob/master/scripts/recipes/install_dpf_plugins.sh

**Comment by guysoft** [comment]
Same happends with glBars

**Comment by falkTX** [comment]
sorry I totally forgot about this.
fixed in e7d0332c71ece4653fa207e2f1476a45fa98e1bf

**Comment by guysoft** [comment]
Yay, thanks.
